modules/issue_tracker/test/issue_tracker_test_plan.md

```diff
@@ -24,10 +25,13 @@ Issue Tracker Edit Existing Issue [Manual Testing]
 5. Submit just a visit label - this should give an error message.
 6. Check that all values are propagated and saved correctly.
 7. Check that watching logging is working - turn it off and on for your current user, and for other watchers on the issue
-
+8. Check that an attachment can be adding to an existing issue.
+9. Test if users assigned to issues can upload attachments.
+10. Test if users can delete their own uploaded attachments.
 
 Permissions [Automation Testing]
 1. Remove access all profile permission.
 2. Remove reporter permission
 3. Remove developer permission
 4. Test that the module behaves correctly as described above. 
+5. Test if user assigned to issue cannot delete attachments of issue owner.
```

Comment:
Is this automated?

Reply:
thanks @johnsaigle, I somehow didn't remove that.